src/install/ai-monitoring/agent-lang/python-config.mdx

```diff
@@ -0,0 +1,103 @@
+---
+componentType: default
+headingText: Configure the Python agent
+freshnessValidatedDate: never
+---
+
+To collect AI data, you need to ensure that certain configurations are enabled when you instrument your AI-powered app. We also recommend updating other variables to improve data retention.
+
+* We recommend applying these configurations in `newrelic.yml`. 
+* If you have a more complex set up with multiple environments, you can configure with environment variables.
+1. Set these variables:
```

Comment:
We talked about this in review, but I'll leave a quick summary here:

- Step is mixing reference and procedural info
- Step should flow a little better in regards to visual presentation of info w/ components
- Rework how the configs are suggestion in regards to default configs and recommended configs